forge-gui/res/cardsfolder/upcoming/archival_whorl.txt

```diff
@@ -0,0 +1,10 @@
+Name:Archival Whorl
+ManaCost:3 U U
+Types:Sorcery
+K:Gift
+SVar:GiftAbility:DB$ MakeCard | Name$ Rhystic Study | Defined$ Promised | GiftDescription$ a Rhystic Study
```

Comment:
Card isn't created
 @Northmoc thoughts ?

Reply:
it works, but you probably want to move it to Battlefield instead...